person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- [ ] Kindly consider adding the `rentals` to attr_accessor instead of just attr_reader, this will make it accessible and writable